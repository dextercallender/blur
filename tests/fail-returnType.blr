int add(int a, int b) {
	int a;
	return "hi";
}

void main() {
	
}