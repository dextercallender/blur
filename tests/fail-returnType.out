Fatal error: exception Failure("incorrect return type")
